cinema/serializers.py

```diff
@@ -114,9 +114,9 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
             ValidationError
         )
         return data
```

Comment:
The validate method in the TicketSerializer is nicely implemented. However, it seems like you're using a class method of the Ticket model inside this method. If this method involves database queries or complex logic, it might be better to move this logic to the serializer or view. This would make your code more maintainable and would keep your models thin.